### ISSUE TITLE
HTTP API UserCredentials

### ISSUE THREAD

**Issue by enumag** [prooph/event-store-client#55]
I don't like passing the `UserCredentials` object to each method that uses the HTTP API (such as SyncProjectionsManager or SyncPersistentSubscriptionsManager). Maybe we could pass a default `UserCredentials` to constructor and use it as fallback?

I can prepare a PR for this if you want. If you don't want this feature then just close this issue.

**Comment by prolic** [comment]
Yes, I'm okay with same. Implementation can be the same as in EventStoreAsyncNodeConnection
